From cd17085c39295c37045d7a942627396997f22943 Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Tue, 7 Oct 2025 18:28:20 +0300
Subject: [PATCH 339/553] QFileSystemModel: remove qpair.h include

Pick-to: 6.8 6.5
Task-number: QTBUG-115841
Change-Id: Iec9e528546f1c6c905e47719aeb85af85c12d3c2
Reviewed-by: Thiago Macieira <thiago.macieira@intel.com>
(cherry picked from commit 08930f58a5093a51fe132236a1f60cad47ce4184)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/gui/itemmodels/qfilesystemmodel_p.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gui/itemmodels/qfilesystemmodel_p.h b/src/gui/itemmodels/qfilesystemmodel_p.h
index 0747b2a07d6..e8599beec2c 100644
--- a/src/gui/itemmodels/qfilesystemmodel_p.h
+++ b/src/gui/itemmodels/qfilesystemmodel_p.h
@@ -22,7 +22,6 @@
 #include <private/qabstractitemmodel_p.h>
 #include <qabstractitemmodel.h>
 #include "qfileinfogatherer_p.h"
-#include <qpair.h>
 #include <qdir.h>
 #include <qicon.h>
 #include <qfileinfo.h>
-- 
2.51.2

